css/css-color/predefined-001.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 4: predefined colorspaces, srgb, decimal values</title>
+<link rel="author" title="Chris Lilley" href="mailto:chris@w3.org">
+<link rel="help" href="https://drafts.csswg.org/css-color-4/#predefined">
+<link rel="match" href="greensquare-ref.html">
+<meta name="assert" content="Color function with explicit srgb value as decimal matches sRGB #008000">
+<style>
+    .test { background-color: red; width: 12em; height: 6em; margin-top:0}
+    .ref { background-color: #008000; width: 12em; height: 6em; margin-bottom: 0}
+    .test {background-color: color(srgb 0 0.5 0)}

```

Comment:
Should this be 0.5019607843137255 (128/255) rather than 0.5?

(And the equivalent for tests 2, 3, and 4?  And then were the numbers in the other tests computed based off of 0.5 as well?)

Reply:
Yes, it should be.

(Better is to use `#33` or a multiple in the hex; that gives an exact `.2` in decimal.)

Reply:
I have redone the tests to use #009900 and 0.6 or 60%

Yes, the others were computed from 0 0.5 0. I have recomputed them from #009900 and rounded to 5 significant figures.

Reply:
By the way, conversions done this time using [this sample app](https://colorjs.io/apps/convert/?color=color(xyz%200.12266%200.22836%200.03093)&precision=5) which uses [color.js](https://colorjs.io/) that Lea and I wrote. It allows setting the precision, and converts any color input into all the color outputs (more than Color 4 and Color HDR support).